lib/syscall_shim/include/uk/syscall.h

```diff
@@ -267,9 +270,22 @@ typedef long uk_syscall_arg_t;
 	{								\
 		long ret;						\
 									\
+		/* Mark system call entry */				\
+		uk_syscallchain_count++;				\
+		uk_sysentertab_run(&(struct uk_sysentertab_ctx){	\
```

Comment:
Consider if you prefer `uk_sysentrytab_*` / `uk_sysexittab_*` or `syscalltab_`entry_*` `sycalltab_exit_*`. Very subjective though, and I'm not the maintainer, so feel free to leave as is if you prefer the current name.

Reply:
Yes this is the old version, I renamed in the first fixup to uk_syscall_entertab/exittab